### PR TITLE
Fixed paper p 1.1 - p. 1.5

### PR DESCRIPTION
**Pull request by Graur**
I've change the EO code in this paper according to the latest EO version. List of changes:
- p. 1.1 (Goto) 
- p. 1.2 (Pointers) 
- p. 1.3 (Procedures) 
- p. 1.5 (Destructors)

### CODE REVIEW

**Comment by Graur** [comment]
@yegor256 Please, have a look

File: paper.tex [hunk 1]
```diff
@@ -108,12 +108,12 @@ \subsubsection{Backward Jump}
     goto
       [g]
         seq > @
-          i.write (i.plus 1)
+          i.lt (i.plus 1)
```
Comment (yegor256) [inline paper.tex]: @Graur this sounds like a typo. are you sure?

File: paper.tex [hunk 2]
```diff
@@ -254,19 +254,21 @@ \subsubsection{Pointers to Data}
...
+pointer. > p1
+  heap 1024
+  0
   108
 [p] > f
   seq > @
-    book (p.plus 7) > b
-    b.title.write "Object Thinking"
+    book (p.add 7) > b
+    b.title.write ("Object Thinking").as-bytes
```
Comment (yegor256) [inline paper.tex]: @Graur this line will be understood as 

```
(b.title.write ("Object Thinking")).as-bytes
```

better use two lines, for clarity

**Comment by yegor256** [comment]
@Graur see above

File: paper.tex (hunk 1 above)
Comment (Graur) [reply paper.tex]: @yegor256 Indeed, it's a typo

**Comment by Graur** [comment]
@yegor256 Can you check again, please?

**Comment by yegor256** [comment]
@Graur thanks!

**Pull request merged**